package.json

```diff
@@ -47,13 +46,13 @@
     "@babel/register": "^7.4.0",
     "@lerna/child-process": "^3.14.2",
     "@lerna/prompt": "^3.13.0",
-    "@wdio/cli": "^5.7.2",
-    "@wdio/dot-reporter": "^5.7.2",
-    "@wdio/jasmine-framework": "^5.7.2",
-    "@wdio/local-runner": "^5.7.2",
-    "@wdio/sauce-service": "^5.7.2",
-    "@wdio/spec-reporter": "^5.7.2",
-    "@wdio/sync": "^5.7.2",
+    "@wdio/cli": "^5.10.10",
```

Comment:
`wdio` packages are updated because of a bug where there were errors thrown because `fibers.node` was not found. So had to update everything.

Reply:
Also the test output looks really good with logs on top and status at bottom.